views/courses/add-tool.hbs

```diff
@@ -7,7 +7,7 @@
 
     {{#content "page"}}
         <section class="section-course">
-                <div class="list-group">
+                <div class="list-group" data-testid="tool_list">
```

Comment:
maybe choose another name as "tool_list" could be mixed up with the tools components. maybe a name fitting the current situation would be good like "course_tool_list" or sth

Reply:
changed